tests/run-multitests.py

```diff
@@ -264,7 +269,7 @@ def run_script(self, script):
             output = self.pyb.exec_(script)
         except pyboard.PyboardError as er:
             err = er
-        return str(output.strip(), "ascii"), err
+        return decode(output), err
```

Comment:
I think you accidentally removed the strip.

Reply:
oops, thanks! Fixed